src/environmentd/src/environmentd/main.rs

```diff
@@ -277,6 +277,9 @@ pub struct Args {
     /// The prefix to prepend to all kubernetes object names.
     #[clap(long, env = "ORCHESTRATOR_KUBERNETES_NAME_PREFIX")]
     orchestrator_kubernetes_name_prefix: Option<String>,
+    /// Whether to enable pod metrics collection.
+    #[clap(long, env = "ORCHESTRATOR_KUBERNETES_DISABLE_POD_METRICS_COLLECTION")]
+    orchestrator_kubernetes_disable_pod_metrics_collection: bool,
```

Comment:
Ugh, I assume we need this to be negative for backwards compatibility. It still feels gross having a negative var, rather than a positive one. I wonder if there is a way to have clap default to true, but provide a flag to set it to false.

Reply:
yeah - that would be ideal but it looked like it would be pretty obnoxious to implement in clap